windows-vm-O365/azuredeploy.json

```diff
@@ -0,0 +1,257 @@
+      }
+    },
+    "vmName": {
+      "type": "string",
+      "metadata": {
+        "description": "Local name for the VM can be whatever you want"
+      }
+    },
+    "vmAdminUserName": {
+      "type": "string",
+      "metadata": {
+        "description": "VM admin user name"
+      }
+    },
+    "vmAdminPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "VM admin password. The supplied password must be between 8-123 characters long and must satisfy at least 3 of password complexity requirements from the following: 1) Contains an uppercase character 2) Contains a lowercase character 3) Contains a numeric digit 4) Contains a special character."
+      }
+    },
+    "vmSize": {
+      "type": "string",
+      "metadata": {
+        "description": "Desired Size of the VM. Any valid option accepted but if you choose premium storage type you must choose a DS class VM size."
+      },
+      "defaultValue": "Standard_DS2"
+    },
+    "vmOSVersion": {
+      "type": "string",
+      "defaultValue": "10-Enterprise-N",
+      "allowedValues": [
+        "10-Enterprise-N",
+        "7.0-Enterprise-N",
+        "8.1-Enterprise-N"
+      ],
+      "metadata": {
+        "description": "Which version of Windows would like to deploy"
+      }
+    },
+    "dnsLabelPrefix": {
+      "type": "string",
+      "metadata": {
+      "description": "DNS Label for the Public IP. Must be lowercase. It should match with the following regular expression: ^[a-z][a-z0-9-]{1,61}[a-z0-9]$ or it will raise an error."
+      }
+    },
+    "officeVersion": {
+      "type": "string",
+      "defaultValue": "Office2016",
+      "allowedValues": [
+        "Office2016",
+        "Office2013"
+      ],
+       "metadata": {
+        "description": "Which version of Office would you would like to deploy"
+      }
+    },    
+    "setupOfficeScriptFileName": {
+      "type": "string",
+      "defaultValue": "DeployO365SilentWithVersion.ps1",
+      "metadata": {
+        "description": "PowerShell script name to execute"
+      }
+    }
+  },
+    "variables": {
+      "storageName": "[concat(uniquestring(resourceGroup().id), 'standardsa')]",
+      "vnet01Prefix": "10.0.0.0/16",
+      "vnet01Subnet1Name": "Subnet-1",
+      "vnet01Subnet1Prefix": "10.0.0.0/24",
+      "vmImagePublisher": "MicrosoftVisualStudio",
+      "vmImageOffer": "Windows",
+      "vmOSDiskName": "VMOSDisk",
+      "vmVnetID": "[resourceId('Microsoft.Network/virtualNetworks', 'Vnet01')]",
+      "vmSubnetRef": "[concat(variables('VMVnetID'), '/subnets/', variables('Vnet01Subnet1Name'))]",
+      "vmStorageAccountContainerName": "vhds",
+      "vmNicName": "[concat(parameters('VMName'), 'NetworkInterface')]",
+      "vmIP01Name": "VMIP01"
+    },
+    "resources": [
+      {
+        "name": "[variables('storageName')]",
+        "type": "Microsoft.Storage/storageAccounts",
+        "location": "[resourceGroup().location]",
```

Comment:
It's probably better to define a variable "location": "[resourceGroup().location]" and then use that as the location for all resources; that way, you can change the location variable, and all resource locations will change with it
 Add a line note